### ISSUE TITLE
«Requirements» | «Description» labels mismatch 

### ISSUE THREAD

**Issue by olea** [openSUSE/osem#1830]
Checked at demo.osem.io:

1. as a user you add a proposal of event, say at /conferences/osemdemo/program/proposals/11/edit and there appears a field called «requirements».
2. as admin you check the proposed event, say at  /admin/conferences/osemdemo/program/events/11 but the field containing the text is tittled «Description».

Check screenshot:

![captura de pantalla de 2017-11-30 18-44-26](https://user-images.githubusercontent.com/141267/33446222-ac931960-d5ff-11e7-9ccb-68dcd4ac8fe9.png)

Guess this is wrong.

**Comment by olea** [comment]
So this is the origin of the confusion at [_proposal_form.html.haml](https://github.com/openSUSE/osem/blob/master/app/views/proposals/_proposal_form.html.haml):

```
    %p.text-right
      = link_to '#description', 'data-toggle' => 'collapse' do
        Do you require something special?
    #description{ class: "collapse #{ 'in' if @event.description.present? }" }
      = f.input :description, input_html: { rows: 5 }, label: 'Requirements', placeholder: 'Eg. Whiteboard, printer, or something like that.'
```

The propierty @event.description should be called something like @event.requirements.

I prefer not to patch it by myself bc not into Ruby :-/

**Comment by olea** [comment]
Fixed in #1859.
